pilot/cmd/pilot-agent/options/options.go

```diff
@@ -80,6 +80,10 @@ var (
 
 	secretRotationGracePeriodRatioEnv = env.Register("SECRET_GRACE_PERIOD_RATIO", 0.5,
 		"The grace period ratio for the cert rotation, by default 0.5.").Get()
+
+	secretRotationGracePeriodRatioJitterEnv = env.Register("SECRET_GRACE_PERIOD_RATIO_JITTER", .1,
```

Comment:
This can skew us by over 2 hours which I think is a bit excessive? Probably +/- 10 min or so is sufficient with the defaults of 24hr TTL? WDYT?

Reply:
Just worried that we end up adding confusion when someone looks at a cert and its very obviously outside of the rotation interval and they think something is wrong. A few minutes is no problem, a few hours maybe causes confusion

Reply:
Makes sense! Made the default `0.01` which comes out to about ~15 minutes over 24 hours.